### PR TITLE
docs: translate API Reference > APIs > `lazy`

### LINKED ISSUE #471: `lazy`
Page: https://id.react.dev/reference/react/lazy

Assignee: @r17x

### PR DESCRIPTION
**Pull request by r17x**
Translation for page https://id.react.dev/reference/react/lazy

Closes #471

### CODE REVIEW

File: src/content/reference/react/lazy.md
Comment (mazipan) [inline src/content/reference/react/lazy.md:35]: Typo sempai, harusnya sampai.
Typo Anda
Nambahin reject, biar tau yang dimaksud menolak itu adalah reject nya Promise.

```suggestion
* `load`: Sebuah fungsi yang mengembalikan [Promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) atau *thenable* lain (sebuah objek yang mirip dengan *Promise* dan memiliki metode `then`). React tidak akan memanggil `load` sampai pertama kali Anda mencoba untuk merender komponen yang dikembalikan. Setelah React pertama kali memanggil `load`, React akan menunggu sampai komponen itu selesai, dan kemudian merender nilai yang telah diselesaikan sebagai komponen React. Baik *Promise* yang dikembalikan maupun nilai yang diselesaikan dari *Promise* akan dicache, sehingga React tidak akan memanggil `load` lebih dari satu kali. Jika *Promise* menolak (reject), React akan `throw` alasan penolakan ke *Error Boundary* terdekat untuk ditangani.
```

File: src/content/reference/react/lazy.md
Comment (mazipan) [inline src/content/reference/react/lazy.md:39]: Typo

```suggestion
`lazy` mengembalikan komponen React yang dapat Anda render di dalam *tree*. Ketika kode untuk komponen *lazy* masih dimuat, mencoba merendernya akan *suspend.* Gunakan [`<Suspense>`](/reference/react/Suspense) untuk menampilkan indikator pemuatan ketika komponen tersebut dimuat.
```

File: src/content/reference/react/lazy.md
Comment (mazipan) [inline src/content/reference/react/lazy.md:65]: Typo
```suggestion
Untuk menunda pemuatan kode komponen ini hingga dirender untuk pertama kalinya, ganti *import* ini dengan:
```

File: src/content/reference/react/lazy.md
Comment (mazipan) [inline src/content/reference/react/lazy.md:75]:
```suggestion
Setelah kode komponen Anda dimuat saat digunakan (*on demand*), Anda juga perlu menentukan apa yang harus ditampilkan ketika dimuat. Anda dapat melakukan ini dengan membungkus komponen *lazy* atau salah satu *parent* ke dalam [`<Suspense>`](/reference/react/Suspense):
```

File: src/content/reference/react/lazy.md
Comment (mazipan) [inline src/content/reference/react/lazy.md:188]:
```suggestion
Jangan deklarasikan `lazy` komponen *di dalam* komponen lain:
```

**Review by mazipan — APPROVED** [review]

File: src/content/reference/react/lazy.md
Comment (resir014) [inline src/content/reference/react/lazy.md:186]:
```suggestion
### komponen `lazy` saya disetel ulang secara tidak terduga {/*my-lazy-components-state-gets-reset-unexpectedly*/}
```

File: src/content/reference/react/lazy.md
Comment (resir014) [inline src/content/reference/react/lazy.md:188]:
```suggestion
Jangan deklarasikan komponen `lazy` *di dalam* komponen lain:
```

File: src/content/reference/react/lazy.md
Comment (resir014) [inline src/content/reference/react/lazy.md:75]:
```suggestion
Setelah kode komponen Anda dimuat saat digunakan (*on demand*), Anda juga perlu menentukan apa yang harus ditampilkan ketika dimuat. Anda dapat melakukan ini dengan membungkus komponen *lazy* atau salah satu induknya ke dalam [`<Suspense>`](/reference/react/Suspense):
```

parent component -> komponen induk

**Review by resir014 — CHANGES_REQUESTED** [review]
Some required changes:

**Comment by r17x** [comment]
Need review from you @resir014

**Review by mhaidarhanif — APPROVED** [review]
It's good. @resir014 can confirm.

**Review by mazipan — APPROVED** [review]

**Comment by r17x** [comment]
Waiting for @resir014 approval

**Review by mazipan — APPROVED** [review]

File: src/content/reference/react/lazy.md
Comment (mhaidarhanif) [inline src/content/reference/react/lazy.md:7]: How about this? Continuing from https://github.com/reactjs/id.react.dev/discussions/337#discussioncomment-5759429

```suggestion
`lazy` memungkinkan Anda menangguhkan (*defer*) pemuatan kode komponen hingga komponen tersebut di-*render* untuk pertama kalinya.
```

**Review by mhaidarhanif — CHANGES_REQUESTED** [review]
Suggestion.

File: src/content/reference/react/lazy.md
Comment (r17x) [reply src/content/reference/react/lazy.md]: Applied @mhaidarhanif

**Review by mhaidarhanif — APPROVED** [review]

**Comment by mhaidarhanif** [comment]
Okay let's wait again for @resir014

File: src/content/reference/react/lazy.md
Comment (resir014) [inline src/content/reference/react/lazy.md:43]:
```suggestion
### Fungsi `load` {/*load*/}
```

File: src/content/reference/react/lazy.md
Comment (resir014) [inline src/content/reference/react/lazy.md:33]:
```suggestion
#### Parameter {/*parameters*/}
```

Suggested change for API reference headings, see: https://github.com/reactjs/id.react.dev/discussions/337#discussioncomment-5769320

File: src/content/reference/react/lazy.md
Comment (resir014) [inline src/content/reference/react/lazy.md:37]:
```suggestion
#### Mengembalikan {/*returns*/}
```

Idem.

File: src/content/reference/react/lazy.md
Comment (resir014) [inline src/content/reference/react/lazy.md:45]:
```suggestion
#### Parameter {/*load-parameters*/}
```

File: src/content/reference/react/lazy.md
Comment (resir014) [inline src/content/reference/react/lazy.md:49]:
```suggestion
#### Mengembalikan {/*load-returns*/}
```

File: src/content/reference/react/lazy.md
Comment (resir014) [inline src/content/reference/react/lazy.md:186]:
```suggestion
### Komponen `lazy` saya disetel ulang secara tidak terduga {/*my-lazy-components-state-gets-reset-unexpectedly*/}
```

**Review by resir014 — CHANGES_REQUESTED** [review]
@r17x some suggested changes according to https://github.com/reactjs/id.react.dev/discussions/337#discussioncomment-5769320

**Comment by r17x** [comment]
Updated @resir014

File: src/content/reference/react/lazy.md
Comment (mazipan) [inline src/content/reference/react/lazy.md:37]:
```suggestion
#### Kembalian {/*returns*/}
```

Comment (r17x) [reply src/content/reference/react/lazy.md:37]: Wait, What I need to follow 😂 

https://github.com/reactjs/id.react.dev/pull/342#discussion_r1181398261

Comment (mazipan) [reply src/content/reference/react/lazy.md:37]: https://github.com/reactjs/id.react.dev/discussions/337#discussioncomment-5769350

Comment (r17x) [reply src/content/reference/react/lazy.md]: Done @mazipan @resir014

**Review by mazipan — APPROVED** [review]
Looks good

**Comment by r17x** [comment]
Need review @resir014

**Review by resir014 — APPROVED** [review]

**Pull request merged**
